CITATION.cff

```diff
@@ -0,0 +1,23 @@
+cff-version: 1.2.0
+message: 'If you use this software, please cite it as below.'
+title: 'Javascript implementation of JCAMP-DX converter'
+abstract: 'This package allows to convert a JCAMP-DX file to a JSON that can be further processed in javascript'
+repository-artifact: http://cheminfo.github.io/jcampconverter/
+repository-code: https://github.com/cheminfo/jcampconverter
+authors:
+  - family-names: 'Miranda'
+    given-names: 'Santiago'
```

Comment:
@santimirandarp could you add (create if required) your orcid ?

Also please could you update title, abstract, KW, url, etc.